From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Mon, 1 Feb 2021 23:10:06 +0000
Subject: DROP ME: Out of tree difference caused by unclean revert

7c0551d428c1 ("Revert "FROMLIST: modpost: Make static exports fatal"")
was reverting more than just the FROMLIST patch. This patch compensates
the difference and should be dropped.

Signed-off-by: Matthias Maennich <maennich@google.com>
Change-Id: Ie1f0ca9054c20f337541baa93193eaa47a0212ff
---
 scripts/mod/modpost.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/mod/modpost.c b/scripts/mod/modpost.c
--- a/scripts/mod/modpost.c
+++ b/scripts/mod/modpost.c
@@ -2647,9 +2647,9 @@ int main(int argc, char **argv)
 
 		for (s = symbolhash[n]; s; s = s->next) {
 			if (s->is_static)
-				error("\"%s\" [%s] is a static %s\n",
-				      s->name, s->module->name,
-				      export_str(s->export));
+				warn("\"%s\" [%s] is a static %s\n",
+				     s->name, s->module->name,
+				     export_str(s->export));
 		}
 	}
 
